### PR TITLE
Changes the order of environment variables

### PR DESCRIPTION
**Pull request by LeMyst**
Changes the order of environment variables to allow the environment variables to be referenced in JAVA_OPTS (like POD_IP in my case)

### CODE REVIEW

**Comment by semihbkgr** [comment]
Hi @LeMyst. Thanks for the PR.
Could you bump the chart version?

**Comment by LeMyst** [comment]
Hi @semihbkgr 
It's done, with a rebase.

File: stable/hazelcast/templates/statefulset.yaml [hunk 1]
```diff
@@ -151,15 +151,15 @@ spec:
         - name: PROMETHEUS_PORT
           value: "{{ .Values.metrics.service.port }}"
         {{- end }}
+        {{- with .Values.env }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
```
Comment (semihbkgr) [inline stable/hazelcast/templates/statefulset.yaml:156]: One last thing, can you apply same changes to the files under `hazelcast-enterprise` and version bump as well.

Comment (LeMyst) [reply stable/hazelcast/templates/statefulset.yaml:156]: Done

**Review by hasancelik — APPROVED** [review]

**Pull request merged**